### PR TITLE
UX improvements

### PR DESCRIPTION
**Pull request by cylwin**

### CODE REVIEW

File: server/controllers/index.coffee [hunk 1]
```diff
@@ -4,6 +4,7 @@ searchEngine = require('../lib/searchEngine')(dataSystem)
 
 #add NPM helpers
 async = require 'async'
+_ = require 'lodash'
```
Comment (doubleface) [inline server/controllers/index.coffee]: No need to use lodash here. Underscore is already available as a the global _.

File: package.json [hunk 2]
```diff
@@ -15,7 +15,8 @@
     "americano": "^0.3.3",
     "async": "^0.9.0",
     "request-json": "^0.5.3",
-    "traceback": "^0.3.1"
+    "traceback": "^0.3.1",
+    "lodash": "^3.10.1"
```
Comment (doubleface) [inline package.json]: No need for lodash

**Comment by doubleface** [comment]
Please make the pullrequest on the dev branch + my small remark and then i will accept it.

File: server/controllers/index.coffee (hunk 1 above)
Comment (cylwin) [inline server/controllers/index.coffee]: Underscore is not available for server files.

Comment (doubleface) [inline server/controllers/index.coffee]: Ooups... my bad

**Comment by doubleface** [comment]
The new pullrequest replaces it

File: server/controllers/index.coffee (hunk 1 above)
Comment (doubleface) [inline server/controllers/index.coffee]: I need to end these hollidays

**Pull request closed without merging**